lib/dul_arclight/traject/ead2_config.rb

```diff
@@ -201,6 +202,9 @@
   accumulator << Arclight::NormalizedTitle.new(title, dates).to_s
 end
 
+to_field 'subarea_sim' do |_record, accumulator, context|
```

Comment:
Seems to me the fields should have the same name on both the collection level and the component level. Here the components have subarea_sim (which is also the facet field) and the collections have subarea_ssm.

Reply:
Actually components don't have a subarea it is a repository/collection thingy.  

The subarea_ssm probably isn't even necessary but I went with the flow of the established pattern of generating the index field, sim, from the saved field, ssm. 

```
to_field 'subarea_ssm', extract_xpath('/ead/archdesc/did/repository/subarea'), strip
to_field 'subarea_sim' do |_record, accumulator, context|
  accumulator.concat context.output_hash.fetch('subarea_ssm', [])
end
```

Probably could of just captured them both with a ssim.

```
<dynamicField name="*_si"    type="string"       stored="false" indexed="true"  multiValued="false" />
<dynamicField name="*_sim"   type="string"       stored="false" indexed="true"  multiValued="true"  />
<dynamicField name="*_ssm"   type="string"       stored="true"  indexed="false" multiValued="true"  />
<dynamicField name="*_ssi"   type="string"       stored="true"  indexed="true"  multiValued="false" />
<dynamicField name="*_ssim"  type="string"       stored="true"  indexed="true"  multiValued="true"  />
```

I'll eliminate the subarea_ssm field since I believe it is unnecessary.